routes/error-tracker.js

```diff
@@ -62,6 +103,24 @@ function getHandler(req, res, next) {
     winston.log('Error', 'Malformed request: ' + params.v.toString(), req);
     return;
   }
+
+  if (ignoreMessageOrException(params.m, params.s)) {
+    res.set('Content-Type', 'text/plain; charset=utf-8');
+    res.status(statusCodes.BAD_REQUEST);
+    res.send('IGNORE\n').end();
+    return;
+  }
+
+  if (params.debug !== '1') {
```

Comment:
If the request doesn't end there, then it assumes you are in production and moves to do everything else.